modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/Layout/LayoutScreen/ObjectLayoutBox.tsx

```diff
@@ -92,18 +92,35 @@ export function ObjectLayoutBox({
 								</ClayButton>
 							)}
 
-							<HeaderDropdown
-								deleteElement={() => {
-									dispatch({
-										payload: {
-											boxIndex,
-											tabIndex,
-										},
-										type: TYPES.DELETE_OBJECT_LAYOUT_BOX,
-									});
-								}}
-								disabled={disabled}
-							/>
+							{type === 'categorization' ? (
+								<HeaderDropdown
+									deleteElement={() => {
+										dispatch({
+											payload: {
+												boxIndex,
+												tabIndex,
+											},
+											type:
+												TYPES.DELETE_OBJECT_LAYOUT_BOX_CATEGORIZATION,
+										});
+									}}
+									disabled={disabled}
+								/>
+							) : (
+								<HeaderDropdown
+									deleteElement={() => {
+										dispatch({
+											payload: {
+												boxIndex,
+												tabIndex,
+											},
+											type:
+												TYPES.DELETE_OBJECT_LAYOUT_BOX,
+										});
+									}}
+									disabled={disabled}
+								/>
+							)}
```

Comment:
Hey @HenriquePereiraa, since the only thing that changes in `dispatch` is the `type`, we can do a ternary condition in the dispatch function to make the code shorter, like:

```tsx

<HeaderDropdown
      deleteElement={() => {
	    dispatch({
		    payload: {
			    boxIndex,
			    tabIndex,
		    },
		    type:
			type === 'categorization'
				? TYPES.DELETE_OBJECT_LAYOUT_BOX_CATEGORIZATION
				: TYPES.DELETE_OBJECT_LAYOUT_BOX,
	    });
      }}
      disabled={disabled}
/>
```